engine/hooks/tasks.go

```diff
@@ -382,16 +382,11 @@ func executeRepositoryWebHook(t *TaskExecution) (*sdk.WorkflowNodeRunHookEvent,
 		payload["git.branch"] = strings.TrimPrefix(pushEvent.Changes[0].RefID, "refs/heads/")
 		payload["git.hash.before"] = pushEvent.Changes[0].FromHash
 		payload["git.hash"] = pushEvent.Changes[0].ToHash
+		payload["git.repository"] = fmt.Sprintf("%s/%s", pushEvent.Repository.Project.Key, pushEvent.Repository.Name)
 
 	default:
-		values, err := url.ParseQuery(t.WebHook.RequestURL)
-		if err != nil {
-			return nil, sdk.WrapError(err, "Hooks> Unable to parse query url %s", t.WebHook.RequestURL)
-		}
-		payload["git.author"] = values.Get("author")
-		payload["git.branch"] = values.Get("branch")
-		payload["git.message"] = values.Get("message")
-		payload["git.hash"] = values.Get("hash")
+		log.Warning("Repository manager not found. Cannotread %s", string(t.WebHook.RequestBody))
```

Comment:
"executeRepositoryWebHook> Repository manager not found. Cannot read %s", string(t.WebHook.RequestBody)"